telemetry/specs/INT.mdk

```diff
@@ -1654,6 +1654,71 @@ INT Metadata Header and Metadata Stack:
    |                   queue occupancy of hop1                     |
    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 `
+## Example with INT including source-only metadata
```

Comment:
I think we should have a separate Appendix where we include examples of Domain specific usage.  I have another I will add to as well.